versions.csv

```diff
@@ -1,9 +1,9 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.36.4
-iothub\service\src\Microsoft.Azure.Devices.csproj, 1.31.1
+iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.36.5
+iothub\service\src\Microsoft.Azure.Devices.csproj, 1.31.2
 shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.27.1
 provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.16.4
-provisioning\transport\amqp\src\Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj, 1.13.5
```

Comment:
Why are you bumping these versions again? We haven't shipped 1.13.5 yet, right?

Reply:
Needs to update his local copy of the lts branch, and rebase on it, I think.

Reply:
https://github.com/Azure/azure-iot-sdk-csharp/commit/53887633adafcee4584b2ef55d118629a4ccf747